backend/public-graph/graph/resolver.go

```diff
@@ -326,19 +335,35 @@ func (r *Resolver) SubmitSessionEvents(ctx context.Context, sessionID int, event
 		})
 	}
 
-	err := r.DataSyncQueue.Submit(ctx, "", messages...)
+	err := r.DataSyncQueue.Submit(ctx, strconv.Itoa(session.ID), messages...)
 	if err != nil {
 		return e.Wrap(err, "failed to submit session events to public worker queue")
 	}
 
 	return nil
 }
 
-func (r *Resolver) AppendFields(ctx context.Context, fields []*model.Field, session *model.Session) error {
+func (r *Resolver) AppendFields(ctx context.Context, chfields []*clickhouse.ClickhouseField, session *model.Session) error {
```

Comment:
we recently introduced `SubmitSessionEvents` as a way to write some fields as special events to clickhouse, right? seems that this `AppendFields` and `SubmitSessionEvents` can be consolidated since what they are doing is very similar (taking fields and writing them to clickhouse, either to `fields` table or to the `events` table). 

maybe we don't need 2 schemas in clickhouse for fields vs. events either?